src/lib.rs

```diff
@@ -11,6 +11,7 @@ mod redismodule;
 pub mod redisraw;
 pub mod redisvalue;
 pub mod stream;
+pub mod digest;
```

Comment:
Nice. It would be good to test out this change. We can do this by using the Digest code added and updating this example module (`data_type.rs`) to implement a callback for the digest function.
https://github.com/valkey-io/valkeymodule-rs/blob/main/examples/data_type.rs

Here is a small example in C where we do the same: https://github.com/valkey-io/valkey/blob/b0f23df16522e91a769c75646166045ae70e8d4e/src/modules/hellotype.c#L296

Reply:
To add a test, we can add a new function for the "alloc" module here: https://github.com/valkey-io/valkeymodule-rs/blob/main/tests/integration.rs

Reply:
Sounds good, added